### PR TITLE
Card: make copy truncation optional

### PR DESCRIPTION
**Pull request by jwijay**
#### What does this PR do?
* makes copy truncation for Card optional (`truncateCopy` option, defaults to `false`).
* allow Accordion Panel to expand full container width (currently `.grommet li` max-width gets applied).

#### What testing has been done on this PR?
Tested against hpe-digitaltoolkit repo in Chrome, Firefox, and Safari.

#### Any background context you want to provide?
Copy truncation was previously desired for Card, but after Stack (which was used by Marquee/Section (toolkit components) was merged with Card, the copy truncation behavior was still applied.

#### What are the relevant issues?
https://github.com/grommet/hpe-digitaltoolkit/issues/137

#### Do the grommet docs need to be updated?
Yes, update Card examples with truncateCopy option.

#### Is this change backwards compatible or is it a breaking change?
Breaking change for previously truncated `<p>` copy in Cards.

### CODE REVIEW

File: src/scss/grommet-core/_objects.accordion-panel.scss [hunk 1]
```diff
@@ -4,6 +4,10 @@
   header:focus {  // too generic, change
     outline: none;
   }
+
+  &.#{$grommet-namespace}list-item {
+    max-width: none;  // allow panel li to expand width fully
```
Comment (ericsoderberghp) [inline src/scss/grommet-core/_objects.accordion-panel.scss]: I just changed the base li styling to not collide with list-item styling. So, I don't think you need this change anymore.

File: src/js/components/Card.js [hunk 2]
```diff
@@ -249,6 +256,7 @@ Card.propTypes = {
     PropTypes.string,
     PropTypes.element
   ]),
+  truncateCopy: PropTypes.bool,
```
Comment (ericsoderberghp) [inline src/js/components/Card.js]: Can we rename this property to just 'truncate'? I think it's semantically reasonable.

Comment (jwijay) [inline src/js/components/Card.js]: @ericsoderberghp: I've updated the PR with the requested changes (removed the list-item styling, and changed the property name from 'truncateCopy' to 'truncate'.

File: src/js/components/Card.js [hunk 3]
```diff
@@ -193,8 +193,15 @@ export default class Card extends Component {
     let link = this._renderLink();
     let videoLayer = this._renderVideoLayer();
 
+    const contentClasses = classnames(
+      {
+        [`${CLASS_ROOT}__content`]: true,
+        [`${CLASS_ROOT}__content--truncated`]: truncate
```
Comment (ericsoderberghp) [inline src/js/components/Card.js]: Can we align the class label with the property name and just call the class `__content--truncate`?

Comment (jwijay) [inline src/js/components/Card.js]: Hi @ericsoderberghp, I've updated the PR to align the class label with the property name.

**Pull request merged**